events/janus_rabbitmqevh.c

```diff
@@ -267,6 +277,59 @@ int janus_rabbitmqevh_init(const char *config_path) {
 	}
 
 	/* Connect */
+	int result=janus_rabbitmqevh_connect();
+	if(result < 0){
+		goto error;
+	}
+
+	/* Initialize the events queue */
+	events = g_async_queue_new_full((GDestroyNotify) janus_rabbitmqevh_event_free);
+	g_atomic_int_set(&initialized, 1);
+
+	GError *error = NULL;
+	handler_thread = g_thread_try_new("janus rabbitmqevh handler", janus_rabbitmqevh_handler, NULL, &error);
```

Comment:
Thread names are 16 characters maximum, please keep it shorter.

Reply:
Style updated